src/components/scroll-wrapper/ScrollWrapper.scss

```diff
@@ -28,31 +28,13 @@ $scroll-shadow-height: 30px;
         &::before {
             top: -$scroll-shadow-height;
             z-index: 20;
-            box-shadow: 0 ($scroll-shadow-height / 2) $scroll-shadow-height (-$scroll-shadow-height / 2.5) rgba(0, 0, 0, .25);
+            box-shadow: 0 2px 8px 0 rgba(0, 0, 0, .1);
```

Comment:
in a previous commit we set the shadow opacity to .12. here it is .1. which is correct?

Reply:
0.10 is correct and documented in Zeplin.